lib/std/os/windows/kernel32.zig

```diff
@@ -452,3 +452,23 @@ pub extern "kernel32" fn RegOpenKeyExW(
 ) callconv(WINAPI) LSTATUS;
 
 pub extern "kernel32" fn GetPhysicallyInstalledSystemMemory(TotalMemoryInKilobytes: *ULONGLONG) BOOL;
+
+pub const LPPROC_THREAD_ATTRIBUTE_LIST = *anyopaque;
+pub const PROC_THREAD_ATTRIBUTE_GROUP_AFFINITY: u32 = 196611;
```

Comment:
Please use the hex values here as the integers are unreadable.
Namespacing them in a struct sounds better to me (pub const PROC_THREAD_ATTRIBUTE = struct { HANDLE_LIST ..}, but I dont have too much of an opinion on this.